media-sound/teamspeak-client/teamspeak-client-3.2.3-r2.ebuild

```diff
@@ -0,0 +1,102 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop toolchain-funcs unpacker xdg-utils
+
+MY_PV="${PV/_/-}"
+
+DESCRIPTION="A client software for quality voice communication via the internet"
+HOMEPAGE="https://www.teamspeak.com/"
+SRC_URI="amd64? ( http://ftp.4players.de/pub/hosted/ts3/releases/${PV}/TeamSpeak3-Client-linux_amd64-${MY_PV}.run )
+	x86? ( http://ftp.4players.de/pub/hosted/ts3/releases/${PV}/TeamSpeak3-Client-linux_x86-${MY_PV}.run )"
+
+KEYWORDS="-* ~amd64 ~x86"
+LICENSE="teamspeak3"
+SLOT="0"
+IUSE="alsa pulseaudio"
+REQUIRED_USE="|| ( alsa pulseaudio )"
+
+RDEPEND="
+	app-arch/snappy:0/1
+	=dev-libs/openssl-1.0*:*
+	dev-libs/quazip
+	dev-qt/qtcore:5
```

Comment:
Do the overrides work fine with old Qt versions?

Reply:
The file with overrides was copied from QtWebEngine master. It had not seen significant changes since it was introduced in version 5.9.2. It does not depend on other parts of Qt apart from the existence of `Qt_5` symbol version which is introduced in the tiny version script used. It does not introduce any functionality, just reexports with a different symbol version, so it should not matter whether the reexport from override or the reexport from Qt is used.

However, I haven't tested that as all my systems are already at Qt 5.12.

Reply:
Yes, it does. I've tested it on my Qt 5.11 system. Works just, as it should, while loading ```overrides.so```.